src/drivers/mct/cime_config/testdefs/testlist_drv.xml

```diff
@@ -23,6 +23,15 @@
       </machine>
     </machines>
   </test>
+  <test name="ERS_C3" grid="f19_g17_rx1" compset="A">
```

Comment:
So does this test now really test the multiple instance functionality for an A compset? I think the answer is yes. We decided that ERS_N3 did not I believe.